locales/en.yml

```
@@ -28,28 +28,39 @@ myprofile:
   posts:
     add: Write a post
   title: My Profile
-chat:
+  chat:
   title: Chat
-  welcome: Start conversation with something nice.
-  with: Chat with {username}
-  dateDiff: "{diff} ago"
+  conversation: 
+      new: Start conversations and they will appear here.
+     welcome: Start conversation with something nice.
```

I would call it `empty` like on other pages

```suggestion
     empty: Start conversation with something nice.
```